web/src/helpers/helpers.ts

```diff
@@ -0,0 +1,64 @@
+import { helpers } from '@vuelidate/validators'
```

Comment:
semicolon

Reply:
I thought we need to add semi-colons for the import section in .vue files only...